client/src/stores/user.ts

```diff
@@ -0,0 +1,31 @@
+import { defineStore } from 'pinia'
+import axios from 'axios'
+
+const baseUrl = 'http://localhost:8081/api'
+
+export const useUserStore = defineStore('user', {
+  state: () => ({
+    authUser: null as string | null
+  }),
+  getters: {
+    user: (state) => state.authUser
+  },
+  actions: {
+    async loginUser(username: string, password: string) {
+      try {
+        await axios
+          .post(`${baseUrl}/auth/token/`, {
+            username: username,
+            password: password
+          })
+          .then((res) => {
+            if (res.status == 200) {
+              this.authUser = username
+            }
+          })
```

Comment:
We will also need to store the token so we can use it to access some of our private endpoints

Reply:
Right, I wasn't sure if we were supposed to store that. The token has two values, a refresh and an access value. I added a token variable that stores both of these in a JSON format.